src/components/views/rooms/MessageComposerButtons.tsx

```diff
@@ -270,6 +270,51 @@ class UploadButton extends React.Component<IUploadButtonProps> {
         );
     }
 }
+
+function showStickersButton(props: IProps): ReactElement {
+    return (
+        props.showStickersButton
+            ? <AccessibleTooltipButton
+                id='stickersButton'
+                key="controls_stickers"
+                className="mx_MessageComposer_button mx_MessageComposer_stickers"
+                onClick={() => props.setStickerPickerOpen(!props.isStickerPickerOpen)}
+                title={
+                    props.narrowMode
+                        ? null
+                        : props.isStickerPickerOpen
+                            ? _t("Hide Stickers")
+                            : _t("Show Stickers")
+                }
+                label={props.narrowMode ? _t("Send a sticker") : null}
```

Comment:
it's a little surprising that we disable the labels and such on narrow mode, but don't do the same to other buttons.

Reply:
Yes, but this change doesn't change anything, so I won't modify that here.